loadtest/control/actions.go

```diff
@@ -375,12 +375,20 @@ func SearchUsers(u user.User) UserActionResponse {
 		return UserActionResponse{Info: "no teams to search for users"}
 	}
 
-	users, err := u.SearchUsers(&model.UserSearch{
-		Term:  "test",
-		Limit: 100,
-	})
-	if err != nil {
-		return UserActionResponse{Err: NewUserError(err)}
+	ticker := time.Tick(time.Duration(rand.Intn(10)) * 1E1 * time.Millisecond)
+	runes := []rune("test")
+	var term string
+	users := make([]*model.User, 0, 100)
+	for i := range runes {
+		<-ticker
+		term += string(runes[i])
+		users, err = u.SearchUsers(&model.UserSearch{
+			Term:  term,
+			Limit: 100,
+		})
+		if err != nil {
+			return UserActionResponse{Err: NewUserError(err)}
+		}
```

Comment:
Looks good. Since we use this in at least two places now and probably a few more in the future maybe we could abstract this *typing behaviour* into a utility function to avoid a lot of repetition. After all, the search term and the API call are the only changing parts between the two actions.

Reply:
I agree on the idea of abstracting the logic and avoid repetition 👍 If we had similar function signature, it would be much easier to do it. Still, I could abstract to something can be considerable.